v2.0/configure-replication-zones.md

```diff
@@ -15,18 +15,36 @@ This page explains how replication zones work and how to use the `cockroach zone
 
 ## Overview
 
-### Replication Zone Levels
+### Pre-Configured Replication Zones
 
-There are four replication zone levels:
+CockroachDB comes with pre-configured replication zones for all user data in the cluster and for some internal system data. These replication zones can be adjusted but not deleted.
```

Comment:
> These replication zones can be adjusted but not deleted.

That's not really true, since you can run `cockroach zone rm` on any of them except for `.default`, and if you do so then the corresponding ranges will just be governed by the `.default` zone.

Reply:
Reworked.